crates/store/re_types/definitions/rerun/blueprint/archetypes/dataframe_query.fbs

```diff
@@ -49,3 +50,38 @@ table DataframeQuery (
     time_range_queries: rerun.blueprint.components.TimeRangeQueries ("attr.rerun.component_optional", nullable,order: 500);
 }
 
+
+
+/// The query for the dataframe view.
+table DataframeQueryV2 (
+    "attr.rerun.scope": "blueprint"
+) {
+    // --- Optional ---
+
+    /// The timeline for this query.
+    ///
+    /// If unset, the time panel's timeline is used and stored.
```

Comment:
```suggestion
    /// If unset, the timeline currently active on the time panel's is used.
```

not sure though what the `and stored` part means here - do mean this is immediately stateful, i.e. the moment the view is spawned this latches? That would be unfortunate imho!